### ISSUE TITLE
Not working at all since last update

### ISSUE THREAD

**Issue by thany** [thehogfather/brackets-code-folding#130]
Since the last update, this plugin doesn't work at all anymore. Nothing. It's as if it just isn't there.

But to prevent a "it doesn't work" issue, I opened up the dev tools to see what's what, and here's what:

```
Exception when calling a 'brackets done loading' handler:
TypeError: Object #<Object> has no method 'on'
    at init (file:///C:/Users/Martijn/AppData/Roaming/Brackets/extensions/user/brackets-code-folding/main.js:297:27)
    at _callHandler (/utils/AppInit.js:93:13)
    at Object._dispatchReady (/utils/AppInit.js:113:13)
    at Object.eval (/brackets.js:289:33)
    at j (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/thirdparty.min.js:559:26676)
    at Object.k.add [as done] (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/thirdparty.min.js:559:26985)
    at Object.d.always (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/thirdparty.min.js:559:27861)
    at eval (/brackets.js:287:30)
    at j (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/thirdparty.min.js:559:26676)
    at Object.k.fireWith [as resolveWith] (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/thirdparty.min.js:559:27489) 
```
I can't help but suspect this is the cause of Code Folding not working...

I'm using Brackets 1.0 on Windows 7 x64. Also tried with *only* Code Folding enabled.

**Comment by thehogfather** [comment]
ok - the reason was due to an upgrade in brackets API. Would you be able to update your brackets to 1.2 or are you using 1.0 for any reason? 

I will update the engine requirements for the plugin.

**Comment by thany** [comment]
I'm using 1.0 because of a [scrolling bug](https://github.com/adobe/brackets/issues/6684) that's holding things back for me...

If anything, can you make it so that 1.0 only installs up to the previous version of the plugin, if you can't make the latest one work on 1.0?

**Comment by thehogfather** [comment]
i've updated the requirements in the registry. Can you uninstall and reinstall?

**Comment by thany** [comment]
Sorry, it doesn't let me install it at all now...

**Comment by thehogfather** [comment]
This issue has  been fixed in @dd7eef9d57ec7e1d4ac4861ca298299ebd6f23e5

**Comment by thany** [comment]
Excellent, seems to be working again. Thanks.
